spatialprofilingtoolbox/apiserver/app/main.py

```diff
@@ -9,24 +9,19 @@
 from fastapi import Response
 from fastapi.responses import StreamingResponse
 
-from spatialprofilingtoolbox.ondemand.counts_service_client import CountRequester
+from spatialprofilingtoolbox.ondemand.service_client import OnDemandRequester
 from spatialprofilingtoolbox.db.exchange_data_formats.study import StudyHandle
 from spatialprofilingtoolbox.db.exchange_data_formats.study import StudySummary
-from spatialprofilingtoolbox.db.exchange_data_formats.metrics import CellFractionsSummary
-from spatialprofilingtoolbox.db.exchange_data_formats.metrics import PhenotypeSymbol
-from spatialprofilingtoolbox.db.exchange_data_formats.metrics import Channel
-from spatialprofilingtoolbox.db.exchange_data_formats.metrics import PhenotypeCriteria
-from spatialprofilingtoolbox.db.exchange_data_formats.metrics import PhenotypeCounts
-from spatialprofilingtoolbox.db.exchange_data_formats.metrics import \
-    ProximityMetricsComputationResult
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import CellFractionsSummary, \
+    PhenotypeSymbol, Channel, PhenotypeCriteria, PhenotypeCounts, \
+    ProximityMetricsComputationResult, SquidpyMetricsComputationResult
```

Comment:
Fair warning, I think I'm going to change this back. I wrote up my case for doing multiple imports on their own line (visual scanning stress-reduction). I am open to being convinced to use the mashed version, if you want to make the case.

Reply:
Maybe we can compromise on the parenthetical version
```
from spatialprofilingtoolbox.db.exchange_data_formats.metrics import (
    CellFractionsSummary,
    PhenotypeSymbol,
    ...
)
```

Reply:
Oh yeah, I was OK with this because it equally well solves the visual scanning problem.

Reply:
And supports the easy-line-permuting functionality.